src/resources/postcss/views/skeleton/_single-event.pcss

```diff
@@ -258,15 +283,26 @@
 		}
 	}
 
-	&.primary + .event-tickets {
+	/* RSVP Block */
+	&.primary + .event-tickets,
+	&.secondary + .event-tickets {
 		margin-top: calc(-1 * var(--spacer-6));
+		margin-bottom: var(--spacer-7);
+		
+		@media (--viewport-medium) {
+			margin-bottom: var(--spacer-8);
+		}
 	}
-
-	&.secondary {
-		margin-top: 0;
-
+	
+	/* Tickets Block */
+	&.primary + .event-tickets + .event-tickets,
```

Comment:
can we add a @todo to move this to ET styles at some point? It doesn't really belong here.
RSVP styles above too.

Reply:
sure, will do